yarn.lock

```diff
@@ -9680,11 +9794,41 @@ node-releases@^1.1.40, node-releases@^1.1.47:
   dependencies:
     semver "^6.3.0"
 
+node-sass@^4.13.1:
```

Comment:
Are you sure you removed `node-sass` and `sass-loader` dependencies? How? Did you just delete them from  package.json?
Do `yarn install` to sync yarn.lock to package.json and next time to remove a package use `yarn remove package1 package2 ...`

Reply:
Just deleted them from package.json but seems I missed yarn.lock. Will fix it in my next commit.

Reply:
Just committed an update to fix the line-height and remove the dependencies.